Packs/Jira/Integrations/JiraEventsCollector/JiraEventsCollector.yml

```
@@ -0,0 +1,63 @@
+category: Analytics & SIEM
+commonfields:
+  id: Jira Events Collector
```

again, sorry about that.
here and everywhere.
```suggestion
  id: Jira Event Collector
```